validation.go

```diff
@@ -292,55 +372,63 @@ loop:
 
 		default:
 			log.Debugf("validation throttled for topic %s", val.topic)
-			throttle = true
-			break loop
+			rch <- validationThrottled
 		}
 	}
 
-	if throttle {
-		v.tracer.RejectMessage(msg, rejectValidationThrottled)
-		return false
-	}
-
+	result := ValidationAccept
 	for i := 0; i < rcount; i++ {
-		valid := <-rch
-		if !valid {
-			return false
+		switch <-rch {
+		case ValidationAccept:
+		case ValidationReject:
+			result = ValidationReject
+			break
+		case ValidationIgnore:
+			if result != validationThrottled {
```

Comment:
@vyzo in the PR comment you said "if any validator ignores or throttles (but none rejects), the message is ignored", but this looks like if a message is both ignored and throttled, the final result will be `validationThrottled` instead of `validationIgnored`. 

That seems fine, I just wanted to mention in case the difference is meaningful. Also, maybe it's worth dropping a code comment here to explain the behavior?

Reply:
yes, this is intentional; throttling supersedes ignoring as we don't know if the throttled validator would have rejected.
I can surely add a comment here nonetheless.

Reply:
done.